packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -43,7 +43,7 @@ export const MarkdownPreview = ({ markdown, append, ...styleProps }: MarkdownPre
     const p: Components['p'] = ({ children, sourcePosition }) => (
       <p>
         {children}
-        {shouldAppend(sourcePosition) && <> {append}</>}
+        {shouldAppend(sourcePosition) && <p> {append}</p>}
       </p>
```

Comment:
I completely forgot what this `append` is about but having a `p` element inside another `p` element is considered incorrect html. As a result react will show a warning in development mode. Is this change really necessary ?